Breadth_First_Search/BFS_in_Csharp.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Collections.Generic;                        // for generic data structures  
+
+namespace GraphBfs
+{
+    class graph
+    {
+       private int vertices;                           // no of vertices in a graph
+       private List<Int32>[] adj;                      //declaraing list
+       public graph(int v)                    
+       {
+           vertices=v;
+           adj=new List<Int32>[v];                     //initialising Adjacency List with number of vertices
+           for(int i=0;i<v;i++)
```

Comment:
Space around operators, and after semi colon.

Reply:
@singh-shreya6 space around operators added